ch_pipeline/core/telescope.py

```diff
@@ -1040,3 +1041,40 @@ def _nearest_freq(tel_freq, map_freq, freq_id, single=False):
     freq_ind = np.nonzero(match_mask)[0]
 
     return freq_ind
+
+
+class MakeTelescope(task.MPILoggedTask):
+    """A simple task to construct a telescope object.
+
+    This removes the need to use driftscan to create a saved beam transfer manager
+    solely for running pipelines that only need the telescope object.
+
+    Attributes
+    ----------
+    telescope_type : str
+        The type of telescope object to create. Generally this must be a fully qualified
+        Python class name, however, \"chime\" (default) can be used to create a
+        `ch_pipeline.core.telescope.CHIME` instance.
+    telescope_config : dict
+        Configuration passed straight to the telescope class via it's `from_config`
+        method.
+    """
+
+    telescope_type = config.Property(proptype=str, default="CHIME")
+    telescope_config = config.Property(proptype=dict)
+
+    def setup(self) -> telescope.TransitTelescope:
+        """Create and return the telescope object."""
+
+        _type_map = {
+            "chime": CHIME,
+        }
+
+        if self.telescope_type in _type_map:
+            tel_class = _type_map[self.telescope_type.lower()]
```

Comment:
This seems a bit redundant - if `telescope_type` and `telescope_type.lower()` aren't the same then this shouldn't work. A "try - except" block or "_type_map.get" would make more sense I think

Reply:
I've just removed the lower casing. Given that this task didn't exist before it's not like there's any legacy behaviour to account for. People should just use the right names.